src/web/app/components/question-types/question-edit-details-form/mcq-field/mcq-field.component.html

```diff
@@ -3,10 +3,10 @@
     <input type="radio" disabled>
   </span>
   <input type="text" class="form-control" [disabled]="!isEditable" [ngModel]="text" #mcq
-         (ngModelChange)="onMcqOptionEntered($event.trim())">
+         (ngModelChange)="onMcqOptionEntered($event.trim())" aria-label="Option">
   <span class="input-group-btn">
     <button type="button" class="btn btn-danger input-group-addon" style="cursor: initial;"
-            tabindex="-1" [disabled]="!isEditable" (click)="deleteMcqOption()">
+            tabindex="-1" [disabled]="!isEditable" (click)="deleteMcqOption()" aria-label="Delete Option">
```

Comment:
```suggestion
           [disabled]="!isEditable" (click)="deleteMcqOption()" aria-label="Delete Option">
```
The delete button should appear in the tab order. Do make the similar changes to the MSQ one as well, and take note of this when encountering similar components in other question types in future PRs.